### ISSUE TITLE
[v2prov] Local RPM Repo Support for installation

### ISSUE THREAD

**Issue by Oats87** [rancher/rancher#36482]
Today, on CentOS 7/8 and RHEL 7/8 systems, a v2prov installation nets a tarball install rather than the RPM install (which is the default when using the RKE2 installation script). This causes issues with selinux.

An approach was brought up to create local RPM repositories on the host, for the purpose of allowing RKE2 to install via RPM. 

This needs two things:
* Install from RPM using a local repo
* Perform "sticky" installs, i.e. a tar ball install remains a tar ball install and vice versa.

**Comment by snasovich** [comment]
This is unscheduled for now unless product comes back and requests this.

**Comment by ron1** [comment]
@snasovich Isn't this required for rke2 selinux support?

**Comment by snasovich** [comment]
@ron1 , I don't believe so. 1-click SELinux support that would include installing RPMs and configuring everything on the nodes is not in the immediate scope of RKE2 provisioning.
It should be possible to set it up manually by following https://rancher.com/docs/rancher/v2.6/en/security/selinux/

@Oats87 , please correct me if I'm wrong here.

**Comment by ron1** [comment]
How do you keep the rke2 selinux rpm version and the current rke2 tarball version in sync during Day 2 upgrades? The rke2 rpm-based installation takes care of this for you.

**Comment by Martin-Weiss** [comment]
This might also be required together with SUC and for SLES (once we have RPM support for SLES)
